linera-sdk/src/test/validator.rs

```diff
@@ -45,7 +50,7 @@ use crate::ContractAbi;
 pub struct TestValidator {
     validator_secret: ValidatorSecretKey,
     account_secret: AccountSecretKey,
-    committee: Committee,
+    committee: Arc<Mutex<(Epoch, Committee)>>,
```

Comment:
Should a validator have a `committee` field at all? In production, it's kind of per chain.

Reply:
This is just a "cached" view of the latest epoch and committee the test is using. Right now all chains should switch epochs in lock-step. This is a limitation and I agree that it should be removed later, but it requires some broader changes.

Reply:
Added #3507 to track this, thanks!